client/components/TestQueueRow/index.jsx

```diff
@@ -337,6 +337,17 @@ const TestQueueRow = ({
     const { status, results } = evaluateStatusAndResults();
     const nextReportStatus = evaluateNewReportStatus();
 
+    const getRowId = tester =>
+        [
+            'plan',
+            testPlanReport.id,
+            'run',
+            currentUserTestPlanRun.id,
+            'assignee',
+            tester.username,
+            'completed'
```

Comment:
Optional nitpick to address: I understand the id's original structure was being preserved with `-completed` being appended. Taking another look again at its use, perhaps something like `-completed-tests` is clearer for the id's purpose, but removing `-completed` altogether also works well and is helpful to reduce some of id's verbosity.